.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "parser": "babel-eslint",
```

Comment:
Why this option necessary?

Reply:
it's not :D
I got by another project, but I will fix it.